tests/sentry/tempest/test_tempest.py

```diff
@@ -0,0 +1,113 @@
+from unittest.mock import patch
+
+from sentry.tempest.tasks import fetch_latest_item_id, poll_tempest_crashes
+from sentry.testutils.cases import TestCase
+from sentry.utils import json
+
+
+class TempestTasksTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.project = self.create_project()
+        self.credentials = self.create_tempest_credentials(self.project)
+
+    @patch("sentry.tempest.tasks.fetch_latest_id_from_tempest")
+    def test_fetch_latest_item_id_task(self, mock_fetch):
+        mock_fetch.return_value = "20001"
```

Comment:
Should this return a json object:
```
{
  "latest_id": "20001"
}
```
Inside `fetch_latest_item_id` we access this field?